tests/manage/z_cluster/cluster_expansion/test_add_non_ocs_workers.py

```diff
@@ -0,0 +1,39 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import tier1, ManageTest
+from ocs_ci.ocs.node import add_nodes, taint_nodes, get_worker_nodes_not_in_ocs
+from ocs_ci.ocs.resources.pod import get_pod_node, get_plugin_pods
+
+logger = logging.getLogger(__name__)
+
+
+@tier1
+class TestAddNonOcsNodes(ManageTest):
+    """
+    Checks for plugin pods toleration come-up on newly added worker nodes
+    """
+
+    def test_add_capacity_node_restart(self, pvc_factory, pod_factory):
+        """
+        1. Add non ocs nodes
+        2. Taint new nodes with app label
+        3. Create app-pods on app_nodes
```

Comment:
4th steps should be 3rd, we need to first check whether our csi node plugins get scheduled on those nodes or not.

Reply:
I also checking that, how does the order of executing makes any difference here ? i create app-pods on non-ocs nodes and then check for plugin pods existence on them. 

Please Let me know if my understanding is wrong here

Reply:
Got it. Incorporated